src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -310,20 +320,49 @@ partial class BoundStateMachineScope
         protected override OperationKind StatementKind => OperationKind.None;
     }
 
-    partial class BoundLocalDeclaration : IVariableDeclarationStatement, IVariable
+    partial class BoundLocalDeclaration : IVariableDeclarationStatement
```

Comment:
It seems odd that a local declaration would be a VariableDeclaration*Statement*... something seems wonky to me here.